FlowersStore/Scenes/FlowersList/FlowersListPresenter.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  FlowersListPresenter.swift
+//  FlowersStore
+//
+//  Created by Rafael Felipe on 30/11/21.
+//
+
+import Foundation
+
+protocol FlowersListPresenterInput: AnyObject {
+    func didTapFavoriteButton()
+    func getFlower() -> Flower
```

Comment:
<!--
  0 failure: 
  1 warning:  Function 'getFlow...
  
  
  DangerID: danger-id-Danger;
  File: FlowersStore/Scenes/FlowersList/FlowersListPresenter.swift;
  Line: 12;
-->

- :warning: Function 'getFlower()' is unused